lib/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -20,6 +20,8 @@ const meta = {
 	url: 'https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties',
 };
 
+const longhandMustValues = ['inherit'];
```

Comment:
[trivial] These suggestions depend on my personal preference. You can ignore them.

- How about using `Set` for a bit more efficiency?
- How about changing to a constant-like name for readability?

```suggestion
const IGNORED_VALUES = new Set(['inherit']);
```

See also:

https://github.com/stylelint/stylelint/blob/d171517376238ff2ba33f98e9e171ee13ca0cd81/lib/rules/color-named/index.js#L27

Reply:
Fixed to use `Set`. https://github.com/stylelint/stylelint/pull/6419/commits/39f4f273aba6354ce2ef8d4245bbf55dbb2c8b0d